banyan/resource_service_web.go

```diff
@@ -286,6 +286,11 @@ func WebSchema() (s map[string]*schema.Schema) {
 			Optional:    true,
 			Description: "access tier group which is associated with service",
 		},
+		"post_auth_redirect_path": {
```

Comment:
@rohitSangamnerkar add corresponding testcase to cover the scenario

Reply:
added test cases